lib/tessel/mdns-list.js

```diff
@@ -0,0 +1,58 @@
+var mdns = require('mdns-js')
```

Comment:
In my opinion, this file shouldn't be in the `tessel` folder because it's not an action that's taken by a single Tessel (unlike erase, deploy, etc.). I think it should be in the `lib` folder.